### PR TITLE
Refactor Tests

### PR DESCRIPTION
**Pull request by mdmedley**
This PR brings the test module into PEP8 compliance and makes
use of the Unittest's native asserts.

### CODE REVIEW

File: tests/test_face_recognition.py [hunk 1]
```diff
@@ -20,65 +20,121 @@
...
+        self.assertEqual(len(encodings), 1)
+        self.assertEqual(len(encodings[0]), 128)
 
     def test_compare_faces(self):
-        img_a1 = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama.jpg"))
-        img_a2 = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama2.jpg"))
-        img_a3 = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama3.jpg"))
-
-        img_b1 = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "biden.jpg"))
+        img_a1 = api.load_image_file(
```
Comment (ageitgey) [inline tests/test_face_recognition.py]: I think using unittest's asserts and the other changes are good, but I personally find splitting these image load statements over 5 lines to be a little too cumbersome. Do you mind switching those back?

Comment (mdmedley) [inline tests/test_face_recognition.py]: @ageitgey Yeah, I can change them back. They're split for PEP8 compatibility but you can define a new max-line-length in your Flake8 configuration

Comment (ageitgey) [inline tests/test_face_recognition.py]: Sure. I'll update the line length config in a separate PR.

File: tests/test_face_recognition.py [hunk 2]
```diff
@@ -20,65 +20,105 @@
 class Test_face_recognition(unittest.TestCase):
 
     def test_load_image_file(self):
-        img = api.load_image_file(os.path.join(os.path.dirname(__file__), "test_images", "obama.jpg"))
-        assert img.shape == (1137, 910, 3)
+        img = api.load_image_file(
```
Comment (ageitgey) [inline tests/test_face_recognition.py]: Can you keep all these load_image_file lines as single lines? Thanks!

Comment (mdmedley) [inline tests/test_face_recognition.py]: Oops! Yeah, changing now

**Pull request merged**